ultralight-java-databind/src/main/java/net/labymedia/ultralight/UltralightDatabindConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package net.labymedia.ultralight;
+
+public final class UltralightDatabindConfiguration {
+    private boolean automaticPrototype;
+
+    private UltralightDatabindConfiguration() { }
+
+    public boolean automaticPrototype() {
+        return automaticPrototype;
+    }
+
+    /**
+     * Create a new <code>UltralightDatabindConfiguration</code> builder.
```

Comment:
Use ``{@link UltralightDatabindConfiguration}`` instead of of ``<code>UltralightDatabindConfiguration</code>``.